util/index.js

```diff
@@ -324,10 +324,14 @@ const getDetailedOrder = (i18n, order, buyer, seller) => {
     const buyerReputation = buyer
       ? sanitizeMD(buyer.total_rating.toFixed(2))
       : '';
+    const buyerTrades = buyer ? sanitizeMD(buyer.trades_completed) : '';
+    const buyerAgeInDays = buyer ? sanitizeMD(buyer.getUserAge) : '';
```

Comment:
```suggestion
    const buyerAgeInDays = getUserAge(buyer);
```
You need to get the real `ageInDays` value as in the suggestion, remember to declare `getUserAge` from `util` like here https://github.com/lnp2pBot/bot/blob/main/bot/ordersActions.js#L11

then add `buyerAgeInDays` and `buyerTrades` as parameter to `order_detail`.